### PR TITLE
first commit - androidctl to its own repo

### PR DESCRIPTION
**Pull request by odra**
First commit with a few changes:
* unit tests (using pytest lib)
* travis integration
* sample readme file - will add sphinx docs + readthedcos integration in another pr

Two followup tasks once it's merged:
* create a release tag.
* change digger-android-sdk-image to install androidctl with pip using the created tag.

Testing it includes installing dependencies and running pytest:

``` 
pip install -U -r requirements.txt
py.test
```

Running `pyhson setup.py install` (sudo) will install the package and `androidctl` cli will become available for usage.

OBS: you will need to run "pip" with sudo if not running pip from pythons virtualenv or docker image.

### CODE REVIEW

**Comment by aliok** [comment]
I don't really see myself Python expert enough to review the whole code and make suggestions.
However, I am ready to test things.

@odra Do you mind some testing instructions?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| Mailing lists:  | [aerogear-users](http://aerogear-users.1116366.n5.nabble.com/) ([subscribe](https://lists.jboss.org/mailman/listinfo/aerogear-users))  |
+|                 | [aerogear-dev](http://aerogear-dev.1069024.n5.nabble.com/) ([subscribe](https://lists.jboss.org/mailman/listinfo/aerogear-dev))  |
+| IRC:            | [#aerogear](https://webchat.freenode.net/?channels=aerogear) channel in the [freenode](http://freenode.net/) network.  |
+
+## Installation
+
+### From source
+
+```
+python setup.pt install
```
Comment (PhilipGough) [inline README.md]: .py

File: androidlib/cli.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  else:
+    pkg.install(name, version)
+
+
+def sync_cmd(path, purge=False, purge_key=True):
+  config = ConfigParser.ConfigParser(allow_no_value=True)
+  try:
+    res = config.read(path)
+  except ConfigParser.Error as err:
+    print 'lalla'
```
Comment (PhilipGough) [inline androidlib/cli.py]: Is this accurate 😆

Comment (odra) [inline androidlib/cli.py]: LOL forgot to remote it :)

**Comment by PhilipGough** [comment]
Reviewed code here and ran tests. All good. Excellent work and nice to get this decoupled

**Pull request merged**